packages/core/src/modules/generic-records/service/GenericRecordService.ts

```diff
@@ -38,6 +38,14 @@ export class GenericRecordService {
     }
   }
 
+  public async deleteById(id: string): Promise<void> {
+    try {
+      await this.genericRecordsRepository.deleteById(id)
+    } catch (error) {
+      throw new AriesFrameworkError(`Unable to delete the genericRecord record with id ${id}. Message: ${error}`)
```

Comment:
In logs I think it would suffice to just call generic records as 'records', for more consistency (e.g. 'Unable to delete record with id ...')